packages/components/package.json

```diff
@@ -64,14 +64,14 @@
   },
   "peerDependencies": {
     "@itwin/appui-abstract": "^4.4.0",
-    "@itwin/components-react": "^4.9.0",
+    "@itwin/components-react": "^5.0.4",
```

Comment:
peerDeps should be set to `^4.9.0 || ^5.0.0` otherwise it's a breaking change.

Reply:
adjusted